### ISSUE TITLE
Unable to start TruckersMP

### ISSUE THREAD

**Issue by RobertJHarnett** [lhark/truckersmp-cli#41]
![Command not working](https://user-images.githubusercontent.com/18248672/63656606-008c7800-c78e-11e9-81f3-5009090ddb50.png)
Trying to get TruckersMP set up for the first time, keeps throwing me this error:

> Proton:** Missing or invalid openvrpaths.vrpath file! [Errno 2] No such file or directory: '/home/robert/.config/openvr/openvrpaths.vrpath'

as well as

> wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"

I'm new to Linux so unsure if this is my doing or an issue with the script.

**Comment by squadramunter** [comment]
Since I also got this problem tho. I recommend you to look at this solution. https://github.com/lhark/truckersmp-cli/issues/34#issuecomment-504429293 This might help for you since it did for me. In your home folder look for a hidden folder called .local and go into the Steam folder. Grom there search for the folder legacycompat and copy all files over to the TruckerMP-CLI directory. I recommend to watch the solution before you ask for help.

**Comment by RobertJHarnett** [comment]
I did have a look and that before posting, but none of the commands worked and couldn't find the folders/files.

While typing this I did notice that the 'Steam' folder under home just has one folder called 'logs' the actual steam folder under home is called '.steam'. I also noticed within truckersmp-cli I dont have a 'Euro Truck Simulator 2'  folder hence why the  `"$XDG_DATA_HOME/truckersmp-cli/Euro Truck Simulator 2/prefix/pfx/drive_c/Program Files (x86)/Steam/"` command wasn't working. Something weird has gone on. Going to just remove any changes and reinstall truckermp-cli.

So I'm missing the same files you were which is what's causing the same error but I wonder if Im also missing some other stuff.

**Comment by squadramunter** [comment]
Did you install Steam first before installing Truckersmp-cli? And what operating system are you using to game on?

**Comment by squadramunter** [comment]
What you can do is open a terminal and install mlocate. After installing mlocate update their database by typing sudo updatedb. And then type mlocate legacycompat. Can you try this for me?

**Comment by Lucki** [comment]
You're likely affected by bug #32 which gets fixed by #33 or the new python port.
Change the wrong folder inside the script to the actual steam path or merge [the listed changes](https://github.com/lhark/truckersmp-cli/pull/33/files) into your local copy.

**Comment by RobertJHarnett** [comment]
I had installed steam and played single player Euro Truck Sim 2  before trying this. I'm using Elementary OS, which I'm pretty sure is built off Ubuntu.

Was typing this while going through solutions, Lucki was spot on with the issue, I merged the listed changes and it works! Thank you both for the timely responses and the help. Also apologies didn't want to be that guy who posted an issue when there was already a thread on the issue, thought my problem was related to #34. Thanks you both once again, no way would I be able to solve that on my own.
